addon/src/-private/sprite.ts

```diff
@@ -1297,7 +1298,30 @@ class CopiedCSS {
   'box-shadow': string;
 }
 
-const COPIED_CSS_PROPERTIES = Object.keys(new CopiedCSS());
+const COPIED_CSS_PROPERTIES = [
```

Comment:
The snippet `Object.keys(new CopiedCSS());` did not work anymore as TS was removing ass unused class properties defined in `CopiedCSS` class.

As far as I understand, this is expected TS behavior hence made the list of props separately from the class interface.